llvm/lib/Transforms/Scalar/PlaceSafepoints.cpp

```diff
@@ -288,6 +288,8 @@ bool PlaceSafepointsPass::runImpl(Function &F, const TargetLibraryInfo &TLI) {
     // with for the moment.
     legacy::FunctionPassManager FPM(F.getParent());
     bool CanAssumeCallSafepoints = enableCallSafepoints(F);
+
+    FPM.add(new TargetLibraryInfoWrapperPass(TLI));
```

Comment:
I think this should probably be split out into a separate preparatory PR, since it is fixing an existing issue as a side effect exposed by the main change here  (iiuc previously too many builtins were available, which was caught by making none available by default).